<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Newtown</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExportStreetMap"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <addaction name="actionViewToolbar_Main"/>
     <addaction name="actionViewToolbar_Tools"/>
    </widget>
    <widget class="QMenu" name="menuFieldVisualization">
     <property name="title">
      <string>Field Visualization</string>
     </property>
     <addaction name="actionViewFieldEnabled"/>
     <addaction name="actionViewFieldContinuous"/>
    </widget>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomReset"/>
    <addaction name="separator"/>
    <addaction name="menuToolbars"/>
    <addaction name="actionViewFieldPainter"/>
    <addaction name="separator"/>
    <addaction name="menuFieldVisualization"/>
   </widget>
   <widget class="QMenu" name="menuMake">
    <property name="title">
     <string>Make</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionRebuild"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSubregions"/>
    <addaction name="separator"/>
    <addaction name="actionBuildComplete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuMake"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRebuild"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomReset">
   <property name="text">
    <string>Zoom Reset</string>
   </property>
  </action>
  <action name="actionViewToolbar_Main">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Main</string>
   </property>
  </action>
  <action name="actionViewFieldPainter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Field Painter</string>
   </property>
  </action>
  <action name="actionViewToolbar_Tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionExportStreetMap">
   <property name="text">
    <string>Export Street Map</string>
   </property>
  </action>
  <action name="actionBuildComplete">
   <property name="text">
    <string>Build Complete</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionRebuild">
   <property name="text">
    <string>Rebuild</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionSubregions">
   <property name="text">
    <string>Subregions</string>
   </property>
  </action>
  <action name="actionViewFieldContinuous">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Continuous</string>
   </property>
  </action>
  <action name="actionViewFieldEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
